### PR TITLE
Allow vgimg font size to be changed

### PR DESCRIPTION
**Pull request by ctessum**

### CODE REVIEW

**Comment by eaburns** [comment]
Interesting. LGTM. How did we not notice that before?

**Comment by sbinet** [comment]
LGTM

**Comment by eaburns** [comment]
Wait. Not LGTM.

File: vg/vgimg/vgimg.go [hunk 1]
```diff
@@ -182,6 +182,7 @@ func (c *Canvas) FillString(font vg.Font, x, y vg.Length, str string) {
 		registeredFont[font.Name()] = true
 	}
 	c.gc.SetFontData(data)
+	c.gc.SetFontSize(float64(font.Size))
```
Comment (eaburns) [inline vg/vgimg/vgimg.go]: font.Size.Points()

**Comment by eaburns** [comment]
Just that one minor thing, then it's OK. (At least I think that it's OK, draw2d.GraphicsContext isn't documented, so I don't know for sure that it takes points.)

**Comment by ctessum** [comment]
fixed

**Comment by eaburns** [comment]
LGTM. Feel free to merge it.

**Comment by kortschak** [comment]
Just a reminder that PRs (particularly like this) need to be squashed and merged ff.

**Comment by kortschak** [comment]
@ctessum, please merge ff. To do this, rebase your change onto master if needed (then push -f to the PR branch to retain a link to the PR discussion), then merge the PR branch into master (ff-only will pick up if you did anything wrong) and push to master.

**Comment by ctessum** [comment]
Ok. I'm relatively new at this; can you explain how what you suggest is different than what I did?

**Comment by kortschak** [comment]
No worries.

GitHub doesn't make it possible to fast forward merges with the web interface. So what you need to do is get all the history organised from the command line and then `git push`.

There is a good outline from @vladimir-ch at https://github.com/gonum/matrix/pull/90

**Pull request merged**